css/premeeting/_premeeting-screens.scss

```diff
@@ -1,31 +1,35 @@
  .premeeting-screen {
-    background: #292929;
-    bottom: 0;
+     position: absolute;
+     left: 0;
+     right: 0;
+     top: 0;
+     bottom: 0;
+  }
+
+ .premeeting-screen {
```

Comment:
Could this declaration be unnecessary addition since we have this class declared on the first line of this file?

Reply:
That is from original jitsi code. 
The idea behind it is to split and highlight absolute positioning of premeeting-screen div.